website/content/en/docs/building-operators/ansible/quickstart.md

```diff
@@ -11,6 +11,13 @@ This guide walks through an example of building a simple memcached-operator powe
 
 - Go through the [installation guide][install-guide].
 - User authorized with `cluster-admin` permissions.
+- An accessible image registry for various operator images (ex. [hub.docker.com](https://hub.docker.com/signup),
+[quay.io](https://quay.io/)) and be logged in in your command line environment.
```

Comment:
If we add that here then instead of example.com we would put quay.io in all.
However, should we add this as a pre-requirement? Could not the person use the docker.io instead? 
Because of this I still think that the best approach is to use `<some-registry>/` and add here the info that is required access to a registry such as quay.io for example.